.buildkite/gpu_pipeline/pipeline.yml

```diff
@@ -1,7 +1,7 @@
 agents:
   queue: clima
   slurm_mem: 8G
-  modules: julia/1.10.0 cuda/julia-pref openmpi/4.1.5-mpitrampoline nsight-systems/2023.4.1
+  modules: julia/1.10.0 cuda/julia-pref openmpi/4.1.5-mpitrampoline nsight-systems/2024.2.1
 
 env:
   JULIA_CUDA_MEMORY_POOL: none
```

Comment:
This line is technically not needed (it is implied by the openmpi module)

Reply:
Thanks! I'll remove it.